Jenkinsfile

```diff
@@ -5,7 +5,7 @@
 pipeline {
   agent { label 'ubuntu-22 && immutable' }
   environment {
-    AWS_ACCOUNT_SECRET = 'secret/observability-team/ci/elastic-observability-aws-account-auth'
+    AWS_ACCOUNT_SECRET = 'secret/observability-team/ci/platform-ingest/aws_account_auth'
```

Comment:
```bash
vault write secret/observability-team/ci/platform-ingest/aws_account_auth \
access_key="REPLACEME" \
csv=@credentials.csv \
secret_key="REPLACEME" \
user="REPLACEME@elastic.co"
```

Reply:
If https://github.com/elastic/beats/blob/main/.buildkite/hooks/pre-command#L9 uses the same format then it could be copied, but I don't have access